src/axom/slic/core/Logger.cpp

```diff
@@ -77,14 +77,75 @@ Logger::~Logger()
   }  // END for all levels
 }
 
+//------------------------------------------------------------------------------
+bool Logger::getAbortFlag(message::Level level)
+{
+  bool ret = false;
+
+  if(level == message::Info || level == message::Debug)
+  {
+    return ret;
+  }
+
+  if(m_isEnabled[level] == false)
+  {
+    return ret;
+  }
+
+  unsigned nstreams = static_cast<unsigned>(m_logStreams[level].size());
+  if(nstreams == 0)
```

Comment:
The nstreams == 0 case is covered by the for-loop below, right? So it can be removed.